frontend/src/services/WpApi.ts

```diff
@@ -3,6 +3,17 @@
 import * as request from 'superagent';
 import config from '../config';
 
+const noCache = function (request) {
+  var timestamp = Date.now().toString();
+  if (request._query !== undefined && request._query[0]) {
+    request._query[0] += '&' + timestamp;
```

Comment:
I don't know the API but this feels a bit hacky :) Cannot `timestamp` be just added to the `_query` array?